src/VisualStudioEditorsSetup/AssemblyRedirects.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio;
+
+[assembly: ProvideEditorsBindingRedirection("Microsoft.VisualStudio.Editors")]
+[assembly: ProvideEditorsBindingRedirection("Microsoft.VisualStudio.AppDesigner")]
```

Comment:
[Here](https://github.com/dotnet/roslyn-project-system/pull/269/files#diff-46e39b3d7102d50a0a09952cdddcc06aL5) it is. Maybe github is not rendering properly for you.